src/server/generic_family.cc

```diff
@@ -786,6 +786,49 @@ OpStatus OpMove(const OpArgs& op_args, string_view key, DbIndex target_db) {
   return OpStatus::OK;
 }
 
+// OpMove touches multiple databases (op_args.db_idx, target_db), so it assumes it runs
```

Comment:
OpMove -> OpCopy

Reply:
Didn't spend time on docs as this was just a draft